docs/elasticsearch-spec.asciidoc

```diff
@@ -100,3 +100,21 @@ Example to create a Kubernetes TLS secret with a self-signed certificate:
 $ openssl req -x509 -newkey rsa:4096 -keyout tls.key -out tls.crt -days 365 -nodes
 $ kubectl create secret tls my-cert --cert tls.crt --key tls.key
 ----
+
+[id="{p}-virtual-memory"]
+=== Virtual memory
```

Comment:
I think the idea behind this doc section was mostly to point out is this setting in the ES spec: https://github.com/elastic/cloud-on-k8s/blob/0fb91b0eaa6f9a4446b216ec44d3790d41e24720/operators/pkg/apis/elasticsearch/v1alpha1/elasticsearch_types.go#L23-L27

It's enabled by default, meaning our init container will take care of tweaking the kernel setting. But it requires the init container to be privileged, hence some users might prefer disabling it.

Reply:
I see, probably I missed the point. From https://github.com/elastic/cloud-on-k8s/issues/970 I thought that we want to ged rid of privileged container and force user to set it...

Reply:
Fixed